src/guides/libraries/c-interop.md

```diff
@@ -253,6 +253,65 @@ To reduce this burden,
 you can use the [`package:ffigen`][ffigen] binding generator
 to automatically create FFI wrappers from C header files.
 
+<a id="native-assets"></a>
+## Building and bundling native assets
+
+{{site.alert.note}}
+  The native assets feature is **experimental**,
```

Comment:
Do the following steps and examples work on Dart 3.2.0 stable or should this mention that a beta or dev build is currently needed?

Reply:
3.2 stable, but with the experiment flag in Dart (or flutter config in Flutter)